doc/conf.py

```diff
@@ -297,3 +298,13 @@
 autosectionlabel_prefix_document = True
 
 linkcheck_ignore = ["https://github.com/PyCQA/pylint/blob/main/pylint/extensions/.*"]
+
+
+# -- Options for pylint_changelog extension ------------------------------------
+
+pylint_changelog_user = "PyCQA"
+pylint_changelog_project = "pylint"
+pylint_changelog_token = os.getenv("GITHUB_TOKEN")
+pylint_changelog_exclude_labels = [
+    "Documentation 📖",
```

Comment:
Regarding this: I said in the original issue "If the PR is labelled maintenance, primer or documentation in github then nothing is required". But I also added 'skip news :mute:', because there's some condition to take into account imo. documentation alone should be skipped but what about documentation and bug together or primer and something else ... ? Maybe just one label with clear expectation is better ? We're doing a lot of maintenance/documentation issue that will need to have one more label if we do that though.

I'm htinking about skip news or  maintenance, primer or documentation and only that label righ tnow.

Reply:
I too think that an explicit "skip news" label would be the cleanest way to go.
Same for the "Other changes" section.